README.md

```diff
@@ -1,21 +1,24 @@
 ---
 name: ChatGPT + Enterprise data
-description: Chat with your data using OpenAI and Cognitive Search.
+description: Chat with your data using OpenAI and AI Search.
 languages:
 - python
 - typescript
 - bicep
 - azdeveloper
 products:
 - azure-openai
-- azure-cognitive-search
+- azure-ai-search
```

Comment:
I assume that this is a known tag now? It has to match what they know about, I believe, or there'll be an error ingesting it into the samples browser.